### ISSUE TITLE
Improve unit test coverage

### ISSUE THREAD

**Issue by terryf82** [Data4Democracy/crash-model#118]

**Comment by terryf82** [comment]
I don't have much experience here so please chime in with opinions.

This will probably spawn several issues but as a starting point
- which specific areas do we think need better coverage?
- how do we determine when we have 'enough' testing (is something like https://coverage.readthedocs.io/en/coverage-4.5.1/ useful)?
- can these tests be incorporated into our github C.I setup?
- what have we tested well so far? what have we tested poorly?

**Comment by j-t-t** [comment]
I really think that everything needs better coverage.  (I'm also of the opinion that every time one fixes a bug, there should be a test added that fails before the test fix and passes after, but that's not something that's reasonable to require)

coverage is a great tool, but how much coverage we'd like is probably up for debate.

Our continuous integration runs py.test, which runs every python test anywhere under src, so any new tests will get run as well.

**Comment by terryf82** [comment]
Working to implement codecov (https://codecov.io/gh/Data4Democracy/crash-model/branch/coverage) but waiting on assistance from D4D member with admin GitHub access.

**Comment by bpben** [comment]
Looks like we're done here.  We have another issue for continuing tracking of codecov #131 .
